iceoryx_posh/include/iceoryx_posh/internal/capro/capro_message.hpp

```diff
@@ -69,9 +69,8 @@ inline log::LogStream& operator<<(log::LogStream& stream, CaproMessageType value
 enum class CaproMessageSubType : uint8_t
 {
     NOSUBTYPE = 0,
-    SERVICE,
-    EVENT,
-    FIELD
+    PUBLISHER,
+    SERVER
```

Comment:
Shall I rename this to `CaproInterfaceType`?

I'll update the changelog once this is decided.

Reply:
I would tend to a more generic name since capro actually doesn't mean anything anymore.
What about `ServiceType` or according to wikipedia it is actually a `MessagingPattern`, see: https://en.wikipedia.org/wiki/Messaging_pattern

Reply:
In this case it should be
```cpp
enum class ServiceType
{
    NO_SERVICE,
    PUBLISH_SUBSCRIBE,
    REQUEST_RESPONSE
};
```
@budrus what do you think?

Reply:
Although, it will be emotionally very hard for me to say goodbye to [CaPro](https://i.pinimg.com/736x/d3/3d/09/d33d0907f5f704eb1a80b996e2900a42.jpg), we should definitely consider it in the long-term.

+1 for `CaproServiceType` for the short-term

Reply:
Done